nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -389,8 +421,25 @@ in
         zfsSupport = true;
       };
 
-      environment.etc."zfs/zed.d".source = "${packages.zfsUser}/etc/zfs/zed.d/";
-      environment.etc."zfs/zpool.d".source = "${packages.zfsUser}/etc/zfs/zpool.d/";
+      services.zfs.zed.settings = {
+        ZED_EMAIL_PROG = mkDefault "${pkgs.mailutils}/bin/mail";
```

Comment:
I believe using `pkgs.system-sendmail` might be appropriate here

Reply:
The provided bash script is expecting the `mail` binary, as the options are not the same for `sendmail` it will not work